src/install/python/python-frameworks/python-agent-cherrypy.mdx

```diff
@@ -0,0 +1,37 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using CherryPy
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
```

Comment:
```suggestion
You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. If using a PasteDeploy with a configuration file see instructions below. Otherwise, there's no specific steps you need to take to integrate with CherryPy so skip straight to the server specific instructions.
```